FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Scheduler/Jobs/AzureBlobJobStore.cs

```diff
@@ -141,8 +141,7 @@ public async Task<bool> CompleteJobAsync(Job job, CancellationToken cancellation
         {
             EnsureArg.IsNotNull(job, nameof(job));
 
-            if (job.Status != JobStatus.Completed
-                && job.Status != JobStatus.Failed)
+            if (job.Status != JobStatus.Completed)
```

Comment:
If we are changing the logics into the new one, we can consider renaming the status from `Completed` to `Succeeded` in future.

Reply:
Sure. That's a good idea. Will change to Succeeded in the next fix.